isis/src/base/objs/PvlKeyword/PvlKeyword.cpp

```diff
@@ -53,6 +53,28 @@ namespace Isis {
     addValue(value, unit);
   }
 
+  /**
+   * Constructs a PvlKeyword object with a name, vector of string values 
+   * and units.
+   * Defaults to unit="".
+   *
+   * @param name The keyword name.
+   * @param vecValue The vector of string values.
+   * @param unit The units the values are given in.
+   */
+  PvlKeyword::PvlKeyword(QString name, vector<string> vecValue,
+                         QString unit) {
+    init();
+    setName(name);
+    QString result = "(";
+    for (int i = 0; i < vecValue.size(); i++) {
+      result += QString::fromStdString(vecValue[i]);
+      if (i != vecValue.size() - 1) result += ", ";
+    }
+    result += ")";
```

Comment:
It would likely be better to run the `addValue` function in the loop rather than build the string. It should be output in the same way when the PvlKeyword is output as a string. Code should look something like:
```
for (int i = 0; i < vecValue.size(); i++) {
  addValue(vecValue[i], unit);
}
```